src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -525,6 +526,8 @@ trait Definitions extends api.StandardDefinitions {
 
     lazy val StringContextClass           = requiredClass[scala.StringContext]
 
+    lazy val ValueOfClass                 = getRequiredClass("scala.ValueOf")
```

Comment:
`getClassIfDefined` to support running this compiler with older standand libraries gracefully.